doc/nrf/documentation/build.rst

```diff
@@ -27,6 +27,37 @@ Complete the following steps to install the required tools:
 
 1. If you have not done so already, install the |NCS| as described in :ref:`auto_installation` or :ref:`manual_installation`.
 #. Install or update all required :ref:`Python dependencies <additional_deps>`.
+#. Install the additional Python dependencies for building the documentation:
+
+   .. tabs::
+
+     .. group-tab:: Windows
+
+        Enter the following command in a terminal window in the :file:`ncs/nrf` folder:
```

Comment:
Should this clarify that the virtual environment needs to be active? Or at least, I presume it should be.

Also, personally, I'd make the folder used either `ncs` (same as is used for the other Python dependencies) or `ncs/nrf/doc` (same as is used for the CMake command). But that's just a matter of taste.

Reply:
@gmarull, should this (step 3- Python dep for the docs/requirements.txt) clarify that the virtual environment needs to be active? For step 2, we will have already made it active.

Reply:
yes, a comment is always good.